test/filesystemevent/main.d

```diff
@@ -113,6 +113,7 @@ class FileCreationTestTask: Task
         this.suspend();
 
         theScheduler.epoll.unregister(inotifier);
+        inotifier.unwatch(this.watched_path.dup);
```

Comment:
Looks like inotifier was not very nicely const-ported :)

Reply:
Yeah, I'll take a look now we got this green how to improve it.